### ISSUE TITLE
High contrast issue on Windows when choosing High Contrast White

### ISSUE THREAD

**Issue by janechu** [microsoft/vscode#109402]
Steps to Reproduce:

1. On a Windows machine, go to https://explore.fast.design/
2. Open up the Windows settings for high contrast
3. Turn on High Contrast
4. Select "High Contrast White" from the dropdown

Does this issue occur when all extensions are disabled?: Yes/No

The background remains in high contrast as the #1E1E1E hex, which puts black text on a dark background.

![image](https://user-images.githubusercontent.com/7559015/97208293-348b7080-1778-11eb-9e0c-652c743494f6.png)

**Comment by aeschli** [comment]
This is about the standalone editor which has a high contrast mode

**Comment by alexdima** [comment]
@janechu While I can reproduce this issue at  https://explore.fast.design/ , I cannot reproduce it at https://microsoft.github.io/monaco-editor/ so I believe there might be something about the CSS on your side that is causing some unwanted behavior and makes me think this is not a generic editor problem. 

After switching to High Contrast White, here is https://microsoft.github.io/monaco-editor/

#### with theme `vs`:
![image](https://user-images.githubusercontent.com/5047891/99502884-fac81700-297d-11eb-91d4-de81e6b5fa27.png)

#### with theme `vs-dark`:
![image](https://user-images.githubusercontent.com/5047891/99502932-09163300-297e-11eb-9c80-61b39ef7350f.png)

#### with theme `high-contrast-dark`:
![image](https://user-images.githubusercontent.com/5047891/99503033-264b0180-297e-11eb-8dad-684e2f3c3e7d.png)
